applications/nrf5340_audio/src/bluetooth/le_audio_cis_headset.c

```diff
@@ -108,9 +108,22 @@ static enum bt_audio_dir caps_dirs[] = {
 #endif
 };
 
+/* Ensure the preferred presentaion delays are within the range of the absolute
+ * and in the maximum >= minimum
+ */
+BUILD_ASSERT(CONFIG_BT_PREFERRED_MAX_PRES_DLY_US >= CONFIG_BT_PREFERRED_MIN_PRES_DLY_US,
```

Comment:
This should be handled in the Kconfig by setting the range of CONFIG_BT_PREFERRED_MAX_PRES_DLY_US to go from CONFIG_BT_PREFERRED_MIN_PRES_DLY_US to some value

Reply:
Ok will change this and the value will be the max pres delay allowed by the system. Also the min preferred delay must be at least the min delay but not greater than the max preferred delay. Will range be able to resolve these?

Reply:
Yeah, I'm thinking:
MIN_PRES_DLY: 4000 (and add comment using the calculations in audio_datapath.h)
MAX_PRES_DLY: 60000 (finger in the air value, but add comment that the range between min and max must include 40000 (spec))
both of these could be prompt-less so that they don't show up in menuconfig

MIN_PREF_PRES_DLY: range MIN_PRES_DLY - 10000 (finger in the air value)
MAX_PREF_PRES_DLY: range MIN_PREF_PRES_DLY - MAX_PRES_DLY

Reply:
Looking at this the CONFIG_BT_PREFERRED_MIN_PRES_DLY_US range should then be from some value to CONFIG_BT_PREFERRED_MAX_PRES_DLY_US which will give a circular definition and hence fail.
I propose that they are ranged by CONFIG_AUDIO_MIN_PRES_DLY_US and CONFIG_AUDIO_MAX_PRES_DLY_US. There need to be a test ad build to ensure the CONFIG_BT_PREFERRED_MIN_PRES_DLY_US not greater than CONFIG_BT_PREFERRED_MAX_PRES_DLY_US. There is a runtime test but maybe that's a bit late then?

Reply:
60ms maybe too large as it could introduce echo or that horrible delay you get over satellite when people keep inadvertently talking over each other. I would suggest we stop at 40ms, which is still high (GSM uses 20ms to avoid these issues) but that is required for broadcast.

Reply:
We don't actually send a timestamp on the mic path so no negotiation of pres delay with the gateway.

Reply:
We shouldn't limit the max presentation delay too much, as this could be just listening to spotify and then the delay has very little importance in terms of a listening experience, but potentially a lot to say about the robustness of the link. In a bidirectional set up it is different of course, but we are not making the end product, so we shouldn't put too many limitations on the customer.

Reply:
Ok provided we stop immediately and drop all decoded buffers, if we do not then there is a possibility that the user may not realise they have pressed pause. I intend having a kconfig value for the max and min so will set it's default to 60ms